chemprop/nn/loss.py

```diff
@@ -1,3 +1,5 @@
+"""Loss functions for use in training models."""
+
```

Comment:
I'd rather stick to the general convention of no module-level docstrings for now. This can change in a future PR though.
```suggestion
```